### DISCUSSION TITLE
[Feature Request]: 剧集订阅优先单集

### DISCUSSION THREAD

**Discussion by xiaobaiwaf** [jxxghp/MoviePilot#3394] [讨论]
### 当前程序版本

v1.5.8

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

希望剧集订阅时可以优先下载单集，洗版时优先合集，目前在某一个电视剧订阅到最后一集时，会下载合集的资源，即使有单集存在
例如今天的一个电视剧《我知道我爱你》，最后一集24集，站点发布了合集和单集，mp下载了合集资源，希望能改进一下！感谢！

**Comment by hermer** [comment]
同需求

**Comment by Griselda-droid** [comment]
同样希望单集优先，而不是拆包处理。或者可以加个开关关闭拆包功能

**Comment by libo26** [comment]
同求！

**Comment by lt19860** [comment]
同求啊，这个功能太需要了
